### ISSUE TITLE
`lipsum_from_seed` missing in 0.9.0

### ISSUE THREAD

**Issue by tgross35** [mgeisler/lipsum#98]
It looks like `lipsum_from_seed` disappeared in 0.9. Is there a way to get this same functionality?

**Comment by tgross35** [comment]
It looks like this was replaced by `lipsum_with_rng`. This is nice but personally, I'd prefer to still also have `lipsum_with_seed` as a convenience method so a user doesn't need to juggle RNGs (not to mention keeping backward compatibility)

**Comment by kalkin** [comment]
Also needless API breakage, without even deprecating the function first. I understand that this crate doesn't promise any stable API, but it's not like this would have been a huge effort.

**Comment by mgeisler** [comment]
Hey folks! Sorry about causing trouble!

@kalkin or @tgross35 would you be up for bringing back a `lipsum_from_seed` function which instantiates a simple RNG for the user?

The function was removed as part of #91, which had the goal of removing the dependency on the system randomness and thus allow lipsum to be used in WASM. There was some discussion there with @reknih about different approaches.

**Comment by tgross35** [comment]
That makes sense about the RNG. Honestly I wouldn't mind just gating the feature behind a `system-rand` flag or something like that. Is that what you're suggesting?

**Comment by reknih** [comment]
A feature like that would work for us! It's good as long as there is a way to use `lipsum` without depending on the system RNG.

**Comment by mgeisler** [comment]
> Honestly I wouldn't mind just gating the feature behind a `system-rand` flag or something like that. Is that what you're suggesting?

Yeah, exactly! We can make the flag a default flag so that people can use the old API from 0.8.0.

I will probably not get around to this anytime soon — @tgross35, could you create a PR for this?

**Comment by kalkin** [comment]
Done. See #99. Feedback welcome.
